README.md

```diff
@@ -10,5 +10,54 @@ http APIs
 <br>
 <br>
 
-## How to Run ##
-Using SBT, run `docker:publishLocal` command to locally publish the docker image. To start the cluster, run `docker-compose up` command.
+## Start Developing Shukra
```

Comment:
Nice! Glad you added this documentation. At some point others will be contributing to this project and this documentation will be great for you to onboard new developers to your project. 👍